src/nvim/ex_docmd.c

```diff
@@ -5311,6 +5333,86 @@ uc_check_code (
     break;
   }
 
+  case ct_MODS:
+  {
+    bool multi_mods = false;
+    typedef struct {
+      bool *varp;
+      char *name;
+    } mod_entry_T;
+    static mod_entry_T mod_entries[] = {
+      { &cmdmod.browse, "browse" },
+      { (bool *)&cmdmod.confirm, "confirm" },
+      { (bool *)&cmdmod.hide, "hide" },
+      { (bool *)&cmdmod.keepalt, "keepalt" },
+      { (bool *)&cmdmod.keepmarks, "keepmarks" },
+      { (bool *)&cmdmod.keepjumps, "keepjumps" },
+      { (bool *)&cmdmod.keeppatterns, "keeppatterns" },
+      { (bool *)&cmdmod.lockmarks, "lockmarks" },
+      { &cmdmod.noswapfile, "noswapfile" }
+    };
+    result = quote ? 2 : 0;
+    if (buf != NULL) {
+      if (quote) {
+        *buf++ = '"';
+      }
+      *buf = '\0';
+    }
+
+    // :aboveleft and :leftabove
+    if (cmdmod.split & WSP_ABOVE) {
+      result += add_cmd_modifier(buf, "aboveleft", &multi_mods);
+    }
+    // :belowright and :rightbelow
+    if (cmdmod.split & WSP_BELOW) {
+      result += add_cmd_modifier(buf, "belowright", &multi_mods);
+    }
+    // :botright
+    if (cmdmod.split & WSP_BOT) {
+      result += add_cmd_modifier(buf, "botright", &multi_mods);
+    }
+
+    // the modifiers that are simple flags
+    for (uint64_t i = 0; i < ARRAY_SIZE(mod_entries); i++) {
+      if (*mod_entries[i].varp) {
+        result += add_cmd_modifier(buf, mod_entries[i].name, &multi_mods);
+      }
+    }
+
+    // TODO(nhooyr): How to support :noautocmd?
+    // TODO(nhooyr): How to support :sandbox
```

Comment:
don't take ownership of upstream TODOs just to satisfy linter. We can just ignore those lint errors.